### ISSUE TITLE
Cloud Commander crashes (JS TypeError) when a client attemps to connect

### ISSUE THREAD

**Issue by pablojrl123** [coderaiser/cloudcmd#276]
After launching Cloud Commander without any arguments:
`` cloudcmd ``
When a client attemps to connect, either the same computer to locahost:8000 or some other machine on the network the following appears in the console:
url: http://localhost:8000/

/usr/local/lib/node_modules/cloudcmd/node_modules/timem/lib/timem.js:31
            callback(error, timeRet);
            ^

TypeError: callback is not a function
    at /usr/local/lib/node_modules/cloudcmd/node_modules/timem/lib/timem.js:31:13

-- END --

It just doesn't make any sense to me, I just installed NodeJS and Cloud Command on the machine. Maybe something is wrong with my PC...
(Tested with NodeJS v10.19.0, v12.16.1 LTS and v13.9.0 Current)

* **Version** (`cloudcmd -v`): v14.3.3
* **Node Version** `node -v`: v13.9.0
* **OS** (`uname -a` on Linux): Linux xubuntu-pc 4.15.0-62-generic #69-Ubuntu SMP Wed Sep 4 20:55:53 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
* **Browser name/version**: Firefox 73.0.1
* **Used Command Line Parameters**: 
* **Changed Config**:

**Comment by coderaiser** [comment]
Fixed with https://github.com/coderaiser/timem/commit/af322ba7227201ca8915d83ccfeb190eb0b071cf, https://github.com/coderaiser/ischanged/commit/cdbce4e3fb26c863fdb390abc56a7756a03c91a6

Please reinstall Cloud Commander. Is it helps?

**Comment by pablojrl123** [comment]
> Fixed with [coderaiser/timem@af322ba](https://github.com/coderaiser/timem/commit/af322ba7227201ca8915d83ccfeb190eb0b071cf), [coderaiser/ischanged@cdbce4e](https://github.com/coderaiser/ischanged/commit/cdbce4e3fb26c863fdb390abc56a7756a03c91a6)
> 
> Please reinstall Cloud Commander. Is it helps?

I removed it with ``npm r cloudcmd -g`` then installed it again with npm (Should I have installed it manually?)
Anyways, now the error is just "Illegal instruction (core dumped)", and the web page shows some 
unformatted text (as shown in the screenshot). This happens with versions 14.3.3, 13.4.2, 12.6.3, and 10.8.2 of Cloud Commander.
![Screenshot from 2020-03-01 12-14-59](https://user-images.githubusercontent.com/41169964/75629223-739bfe80-5bb6-11ea-80f9-4297444dd224.png)
I have used Cloud Commander in other devices and it works flawlessly. I just installed it in a Linux Mint computer (last cloudcmd version and last nodejs version) and it works. Maybe the problem is with my machine, it is a little bit old but running the last Xubuntu/Ubuntu LTS version. Later I'm going to try to reinstall the OS to see if it helps. Thank you for the help anyways :)

**Comment by pablojrl123** [comment]
Well, I don't know much about Node or JS, but after some research about the "Ilegal instruction" error I found out that I has something to do with processor capabilities. Then, I went ahead to NodeJS downloads page, downloaded the source code for the latest LTS version, and compile it in the machine I want to run NodeJS on. And sure enough, cloudcmd works! After all, it is a machine about 12 years old, so the pre-built binaries don't work as expected.

I'm very sorry that I opened an Issue without proper research of my problem.
